src/hooks/useHighlight.ts

```diff
@@ -22,7 +23,7 @@ export const useHighlight = (): Result => {
         if (mounted.current) {
           setHighlight(false)
         }
-      }, 5000)
+      }, HIGHLIGHT_TIME)
```

Comment:
While not strictly necessary with timeouts, as they clear themselves in time :stuck_out_tongue: , a good practice is to cleanup on unmount and when retriggering, something like:
```
const [highlight, setHighlight] = useState(false)
  const mounted = useRef(true)

  const timeoutId = useRef()
  
  const triggerHighlight = (): void => {
    clearTimeout(timeoutId.current)
    if (mounted.current) {
      setHighlight(true)
      timeoutId.current = setTimeout(() => {
          setHighlight(false)
      }, HIGHLIGHT_TIME)

useEffect(() => => {
    mounted.current = false
    clearTimeout(timeoutId.current)
  }, [])